src/views/home/sections/Landing.tsx

```diff
@@ -14,10 +14,7 @@ function Landing() {
 				<Button className="btn-museum btn-landing" href="/apply">
 					Apply as a Hacker
 				</Button>
-				<Button
-					className="btn-museum btn-landing"
-					href="https://forms.gle/tcnikpj5gHnGPNvx7"
-				>
+				<Button className="btn-museum btn-landing" href="/mentor">
```

Comment:
These rewrites aren't working on my end. Is it supposed to normally redirect to the google form page?

Reply:
Did you restart the development server when pulling these changes? If you didn't, it should tell you
> Found a change in next.config.js. Restart the server to see the changes in effect.

Reply:
Oop, looks like I didn't, thanks Taesung.